src/store/index.js

```diff
@@ -16,6 +16,7 @@ export default () => {
                 FacetSelections: {}
             },
             extendedSearchParams: {},
+            keywordUrlParam: null,
```

Comment:
Setting an additional field for the keyword is not needed. The keyword is available from 'searchOutput'. Map it in the SearchBox component and modify the watcher.